go-controller/pkg/node/node_dpu_test.go

```diff
@@ -164,7 +164,7 @@ var _ = Describe("Node DPU tests", func() {
 		It("Fails if dpu.connection-details Pod annotation is not present", func() {
 			pod.Annotations = map[string]string{}
 			fakeClient := newFakeKubeClientWithPod(&pod)
-			podNamespaceLister.On("Get", mock.AnythingOfType("string")).Return(pod, nil)
```

Comment:
@dcbw based on commit d39716645fe8bc7f021e7ea0b5f61bbcf421fe8a ... can you please confirm this change is okay by you?